src/TrueLayer/Payouts/Model/GetPayoutsResponse.cs

```diff
@@ -69,6 +69,15 @@ public record Authorized : PayoutDetails;
         [JsonDiscriminator("successful")]
         public record Successful(DateTime SucceededAt) : PayoutDetails;
 
+        /// <summary>
+        /// Represents a payout that has been executed.
+        /// For open loop payouts this state is terminate. For closed-loop payouts, wait for Settled.
```

Comment:
In both cases, executed is a terminal state for payouts. There's no such thing as a settled payout as we can't know whether it has settled or not in the end user's account.

Reply:
Comment fixed in f640bba.